docs/sphinx/source/whatsnew/v0.9.6.rst

```diff
@@ -30,6 +30,9 @@ Enhancements
   the option of controlling what NSRDB endpoint is used. (:pull:`1736`)
 * :py:func:`pvlib.iotools.get_psm3` now uses the new NSRDB 3.2.2 endpoint for
   hourly and half-hourly single-year datasets. (:issue:`1591`, :pull:`1736`)
+* :py:mod:`pvlib.spa`, and :py:func:`pvlib.solarposition.get_solarposition`/
+  :py:meth:`pvlib.location.Location.get_solarposition` with ``method='nrel_numpy'``
+  or ``method='nrel_numba'``, are now 50-100% faster than before. (:pull:`1748`)
```

Comment:
There's also `pvlib.solarposition.spa_python` and `ModelChain.run_model*`. Maybe less is more in this situation: `The default solar position algorithm (NREL SPA) is now 50-100% faster.`